jasmine/build/bundle.js

```diff
@@ -14,61 +11,139 @@ module.exports=books = [
   }
 ]
 
+},{}],2:[function(require,module,exports){
+module.exports=bookempty = []
+
 },{}],3:[function(require,module,exports){
-/* Test Setup */
-const myInvertedIndex = new InvertedIndexClass();
-const book = require('./../books.json');
-const emptyBook = require('./../bookempty.json');
+module.exports=books3 = [
+  {
+    "title": "Alice in Wonderland",
+    "text": "Alice falls into a rabbit hole and enters a world full of imagination."
+  },
+
+  {
+    "title": "The Lord of the Rings: The Fellowship of the Ring.",
+    "text": "An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring."
+  },
+  {
+    "title": "The Lord of the Rings: The Fellowship of the Ring.",
+    "text": "An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring."
+  }
+]
+
+},{}],4:[function(require,module,exports){
+module.exports=bookwrongformat = [
+  {
+    "what": "Alice in Wonderland",
+    "next": "Alice falls into a rabbit hole and enters a world full of imagination."
+  },
+
+  {
+    "are": "The Lord of the Rings: The Fellowship of the Ring.",
+    "how": "An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring."
+  }
+]
 
-myInvertedIndex.files['book.json'] = book;
+},{}],5:[function(require,module,exports){
+module.exports=notBook = {
+  "title": "hello",
+  "text": "hello"
+}
+},{}],6:[function(require,module,exports){
+'use strict';
 
+var _book = require('./../book.json');
```

Comment:
I thought `require('../books.json')` would be sufficent here or is this a style requirement?